exampleSite/config.toml

```diff
@@ -190,3 +190,9 @@ url = "projects/"
 name = "Contato"
 weight = 5
 url = "contact/"
+
+# Enable html support inside markdown
+[markup]
```

Comment:
Everything looks fine. I'm only against this addition. This is not a mandatory configuration for this theme.
Please, could you remove it?

Reply:
Removing it will not render the blog posts provided correctly.

Reply:
Which one?

Reply:
- This section doesn't work as intended. https://hugo-coder.netlify.app/posts/markdown-syntax/#other-elements--abbr-sub-sup-kbd-mark
- This section doesn't show the photo. https://hugo-coder.netlify.app/posts/placeholder-text/#vagus-elidunt
- And, this doesn't show emoji at all. https://hugo-coder.netlify.app/posts/emoji-support/

Reply:
Yes, I met the same problem thought it is not a big deal.

Reply:
I forgot to come back here.
- About the 'abbr, sub, sup, kbd, mark' no styling, this should be just missing CSS rules.
- The SVG file should be added with a shortcode. I don't know if there is a default one with Hugo, if not I can build one later.
- Emoji should be working. Maybe we are using an old version of Hugo to build the preview. I'm going to take a look on that issue.